server/player/player.go

```diff
@@ -1682,6 +1698,13 @@ func (p *Player) Tick(current int64) {
 			}
 		}
 	}
+
+	if p.s == session.Nop {
```

Comment:
Should use `(*player.Player).session()` here to prevent race conditions

Reply:
Fixed.